src/drive/web/modules/views/Sharings/FilesViewerSharings.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { useQuery } from 'cozy-client'
+import { useRouter } from 'drive/lib/RouterContext'
+import { buildSharingsQuery } from 'drive/web/modules/queries'
+import { getCurrentFileId } from 'drive/web/modules/selectors'
+import { connect } from 'react-redux'
+import FilesViewer, {
+  FilesViewerLoading
+} from 'drive/web/modules/viewer/FilesViewerV2'
+import SharedDocuments from 'cozy-sharing/dist/components/SharedDocuments'
+
+const FilesViewerWithQuery = ({ sharedDocumentIds, ...props }) => {
```

Comment:
Nit: SharingsFileviewer or FilesViewerSharings to keep the same name as the file ?

Nit: Can you add a docstring ?

Reply:
Forgot to renameit!